examples/custom-resources/certmanager/self-signed.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: sandbox
```

Comment:
we never used `sandbox` before, should we use `nginx-ingress`?

Reply:
I left it as `sandbox` as per the cert-manager standard, but I'm open to changing it. I don't think issuers should be deployed in the `nginx-ingress` namespace though, as they are independent from NIC. What do you think?

Reply:
yeah maybe not `nginx-ingress` you're right